### ISSUE TITLE
just a quick question no issue :)

### ISSUE THREAD

**Issue by BlackThief1989** [Healix/Gw2Launcher#115]

**Comment by Healix** [comment]
No. The game doesn't support automatically loading a character, so it would require sending a key press to the game, which is technically against the rules.

**Comment by BlackThief1989** [comment]
Healix , thank you man for the fast responce <3
